public/angular/translations/locale-en.json

```diff
@@ -19,6 +19,8 @@
   "rss": "RSS",
   "elmo": "Elmo",
   "facebook": "Facebook",
+  "smsgh": "SMSGh",
+  "Smsgh": "Smsgh",
```

Comment:
Can we have one consistent way to write this everywhere? What's the official capitalization?